File: diemBFT/src/main/java/diem/model/ProposalMessage.java
package diem.model;

import lombok.AllArgsConstructor;

@AllArgsConstructor
public class ProposalMessage {
    public Block block;
    public TimeoutCertificate lastRoundTC;
    public QuorumCertificate highCommitQC;
    public Signature signature;
    public Author sender;


    public ProposalMessage(Block b, TimeoutCertificate last_tc, QuorumCertificate highCommitQC) {
    }
}

File: diemBFT/model/proposal_message.py
class ProposalMessage:
    def __init__(self, block, last_round_tc, high_commit_qc):
        self.block = block
        self.last_round_tc = last_round_tc
        self.high_commit_qc = high_commit_qc
        # self.signature = signature
        # self.sender = senders